### ISSUE TITLE
Event Label resets with every pageview Avent Action stays at 5

### ISSUE THREAD

**Issue by Larsinholo** [robflaherty/riveted#36]
Hey there,

I installed riveted and saw that with every pageview the 5 sec timer resets (Event Label) and that the event Action stays at 5 ;-( 
Is that the way it is supposed to work? I wanted to see how long in ONE SESSION the people are engaged or is that not possible?

Thanksin advance for your answers!

Regards Lars

**Comment by robflaherty** [comment]
Yep, it keeps track of engaged time on the current page only, and resets between pages. It's not possible to report session-level data to GA. But if you want to segment by session length you can just use GA's session duration metric. If you have Riveted installed the session duration will reflect the engaged time.
